hack/start-flotta.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# arguments: $1 = operator path ; $2 = device-worker path ; $3 = yggdrasil path ; $4 = image registry ; $5 = target
```

Comment:
Overall, I'm ok with this PR (great job! :) ). 
My last doubt is about positional argument. It's easy to wrongly insert values but on the other hand the script will be more complicated and probably It will be an overkill.

Reply:
I would use arguments like in this script:
https://github.com/project-flotta/flotta-scale-tests/blob/main/scripts/run_test_plan.sh

Reply:
Agreed. That's why I added to `docs/user-guide/running.md` examples for each use case so the user will easily copy & paste the command and then change the values in place. WDYT?

Reply:
Done, @gciavarrini @tupyy  WDYT?